transifex/utils.py

```diff
@@ -297,7 +297,7 @@ def merge_pr(self):
                         'Failed to merge [%s/#%d], because of the exception [%s]',
                         self.name, self.pr.number, e,
                     )
-                    return False
+                    raise RuntimeError('Failed to merge.')
```

Comment:
It might be good to specify that the failure was due to a GithubException, and maybe include the exception message?

Reply:
github exception is already being shown by logger.info so no need to show it again i think.

Reply:
It might be good to log that at the error level instead of info, so it's clear in the log that it's an error.